internal/configure.go

```diff
@@ -70,8 +69,8 @@ type InvalidConcurrencyValueError struct {
 	error
 }
 
-func NewInvalidConcurrencyValueError(value int) InvalidConcurrencyValueError {
-	return InvalidConcurrencyValueError{errors.Errorf("Concurrency value is expected to be positive but is: %v", value)}
+func NewInvalidConcurrencyValueError(concurrency string, value int) InvalidConcurrencyValueError {
```

Comment:
Is a type of setting "concurency" is a string? Or there should be the name of any setting?

Reply:
In the first case, let's make a comment.

Reply:
In the second case. It should be more common Error and be placed in a different package.